rules-reviewed/jboss-eap/websphere/tests/WebSphereMQ.windup.test.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/schema/jboss-ruleset">
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../WebSphereMQ.windup.xml</rulePath>
+    <ruleset>
+        <metadata></metadata>
+        <rules>
+            <rule>
```

Comment:
<?xml version="1.0" encoding="UTF-8"?>
<!--
    Creates IDs of the rules.
-->
<xsl:stylesheet version="1.0"
    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
    xmlns:wu="http://windup.jboss.org/schema/jboss-ruleset"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
>
    <xsl:output method="xml"/>


    <!--More specific template for Node766 that provides custom behavior -->
    <xsl:template match="wu:rule">
        <xsl:copy>
            <xsl:attribute name="id"><xsl:number format="01"/>00</xsl:attribute>
            <xsl:apply-templates select="@*[local-name() != 'id']|node()"/>
        </xsl:copy>
    </xsl:template>

    <!-- Copy all. -->
    <xsl:template match="wu:ruleset">
        <xsl:copy>
            <xsl:apply-templates select="@*|node()"/>
        </xsl:copy>
    </xsl:template>

    <!-- Copy all. -->
    <xsl:template match="@*|node()">
        <xsl:copy>
            <xsl:apply-templates select="@*|node()"/>
        </xsl:copy>
    </xsl:template>

</xsl:stylesheet>